twitcher/owsproxy.py

```diff
@@ -160,25 +159,11 @@ def owsproxy_view(request):
         service = request.owsregistry.get_service_by_name(service_name)
     except Exception:
         return OWSAccessFailed("Could not find service")
-    if verify_request(request, service) is False:
+    if request.verify_ows_request is False:
```

Comment:
naming is not intuitive as it feels like a function name

Reply:
done ... but shortened to `is_verified`.